File: URI_1038_snack.java
import java.text.DecimalFormat;
import java.util.HashMap;
import java.util.Scanner;

public class URI_1038_snack {

	public static void main(String[] args) {
		HashMap<Integer, Float> items = new HashMap();
		items.put(1, 4.0f);
		items.put(2, 4.5f);
		items.put(3, 5.0f);
		items.put(4, 2.0f);
		items.put(5, 1.5f);

		Scanner scanner = new Scanner(System.in);
		int itemCode = scanner.nextInt();
		int amount = scanner.nextInt();

		double total = items.get(itemCode) * amount;
		
		String formattedTotal = new DecimalFormat("0.00").format(total);
		System.out.println("Total: R$ " + formattedTotal);
	}
}

File: URI_1038_snack.js
const lines = ["2 3"];

const items = new Map();
items.set(1, 4);
items.set(2, 4.5);
items.set(3, 5);
items.set(4, 2);
items.set(5, 1.5);

const inputArr = lines[0].split(" ");
const itemCode = Number(inputArr[0]);
const amount = Number(inputArr[1]);

const total =  items.get(itemCode) * amount;

console.log(`Total: R$ ${total.toFixed(2)}`);